app/main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __str__(self) -> str:
+        return f"Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}"
```

Comment:
Avoid using line breakers when using multiple strings, return a string in `()`. It will allow you to omit using `\`.